internal/shellgen/tmpl/flake_remove_nixpkgs.nix.tmpl

```diff
@@ -3,9 +3,14 @@
 
    inputs = {
      nixpkgs.url = "{{ .NixpkgsInfo.URL }}";
+
      {{- range .FlakeInputs }}
      {{.Name}}.url = "{{.URLWithCaching}}";
      {{- end }}
+
+     {{- if .PatchGlibc }}
+     nixpkgs-unstable.url = "nixpkgs";
```

Comment:
Make this a hard coded reference.

Reply:
As discussed:

avoid creating `nixpkgs-unstable` and just use `nixpkgs` from above. Just set `nixpkgs` to be the hard coded commit you want in go code.